aspnetcore/fundamentals/target-aspnetcore.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Use ASP.NET Core APIs in a class library
+author: scottaddie
+description: Learn how to use ASP.NET Core APIs in a class library.
+ms.author: scaddie
+ms.custom: mvc
+ms.date: 12/03/2019
+no-loc: [Blazor]
+uid: fundamentals/target-aspnetcore
+---
+# Use ASP.NET Core APIs in a class library
+
+By [Scott Addie](https://github.com/scottaddie)
+
+This document provides guidance for using ASP.NET Core APIs in a class library. For all other library guidance, see [Open-source library guidance](/dotnet/standard/library-guidance/).
+
+## Use the ASP.NET Core shared framework
+
+With the release of .NET Core 3.0, many ASP.NET Core packages are no longer published to NuGet. Instead, the packages are included in the `Microsoft.AspNetCore.App` shared framework, which is installed with the .NET Core SDK. The shared framework was previously distributed as a NuGet package. For a list of packages no longer being published, see [Remove obsolete package references](xref:migration/22-to-30#remove-obsolete-package-references).
+
+As of .NET Core 3.0, projects using the `Microsoft.NET.Sdk.Web` MSBuild SDK implicitly reference the shared framework. Projects using the `Microsoft.NET.Sdk` or `Microsoft.NET.Sdk.Razor` SDK must target ASP.NET Core to use ASP.NET Core APIs in the shared framework.
+
+To target ASP.NET Core, add the following `<FrameworkReference>` element to your project file:
+
+[!code-xml[](target-aspnetcore/samples/single-tfm/netcoreapp3.0-basic-library.csproj?highlight=8)]
+
+Targeting ASP.NET Core in this manner is only supported for projects targeting .NET Core 3.0.
```

Comment:
"**Referencing** ASP.NET Core in this manner is only supported for projects targeting .NET Core 3.0."